core/iwasm/aot/aot_loader.c

```diff
@@ -264,6 +354,15 @@ load_string(uint8 **p_buf, const uint8 *buf_end, AOTModule *module,
     if (str_len == 0) {
         str = "";
     }
+#if (WASM_ENABLE_WORD_ALIGN_READ != 0)
+    else if (is_vram_word_align) {
+        if (!(str = const_str_set_insert((uint8 *)p, str_len, module,
```

Comment:
Does it mean that we will create a new string for the vram mode, no matter the original is terminated with '\0' or not? Is it due to we can not access a string in the vram like normal mode?